mutiny-core/src/node.rs

```diff
@@ -1158,7 +1158,8 @@ impl<S: MutinyStorage> Node<S> {
 
                         if lsp_invoice.payment_hash() != invoice.payment_hash()
                             || lsp_invoice.recover_payee_pub_key() != lsp_pubkey
-                            || (lsp_invoice.amount_milli_satoshis() != Some(amount_sat * 1_000)
+                            || (lsp_invoice.amount_milli_satoshis().map(|s| s / 1_000)
```

Comment:
is there not just an `amount_sats` ? What if it rounds the wrong way?

Reply:
rust always rounds down, ldk only has a msats option

Reply:
changed to just check within 10 sats